plugins/flashrom/flashrom.quirk

```diff
@@ -11,52 +11,8 @@ VersionFormat = quad
 [b8cf5af6-8a46-5deb-ac01-a35b1ea5fb48]
 Plugin = flashrom
 
-# StarLite Mk II (HwId - AMI)
-[013b60e5-1023-5bee-8ae5-14cae21377b7]
```

Comment:
Don't you need this still? It's not coreboot

Reply:
Nope :)